Configuration/AlCa/python/autoCond.py

```diff
@@ -90,13 +90,13 @@
     # GlobalTag for MC production with realistic conditions for Phase1 2023 detector for Heavy Ion
     'phase1_2023_realistic_hi'     :    '140X_mcRun3_2023_realistic_HI_v3',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2024
-    'phase1_2024_design'           :    '140X_mcRun3_2024_design_v1',
+    'phase1_2024_design'           :    '140X_mcRun3_2024_design_Candidate_2024_02_24_11_26_58',
```

Comment:
@cms-sw/alca-l2 I created here GT candidates for testing purposes, please provide a final GT to be included here (and below).

Reply:
Done in another PR https://github.com/cms-sw/cmssw/pull/44078